### PR TITLE
TWC-NL, but it's only NL content

### PR DESCRIPTION
**Pull request by b-n**
These are a couple of changes for the site to make it for our local chapter.

Each commit is somewhat atomic and hopefully describes what was changed and why :-)

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -18,10 +24,10 @@ connect:
   email: Your Email
   name: Your Full Name
   public_link: Social media link
-  public_link_placeholder: Linkedin, Xing, Twitter etc...
+  public_link_placeholder: LinkedIn, Xitter, Mastadon, etc...
```
Comment (chobeat) [inline _i18n/en.yml:27]: typo

File: _i18n/nl.yml [hunk 2]
```diff
@@ -74,24 +79,24 @@ nl:
...
-      - December
+      - januari
+      - februari
+      - maart
+      - april
+      - mei
+      - juni
+      - juli
+      - augustus
+      - septermber
```
Comment (chobeat) [inline _i18n/nl.yml:99]: typo

File: _i18n/nl.yml [hunk 3]
```diff
@@ -3,46 +3,51 @@ langs:
...
-  public_link_placeholder: LinkedIn, Twitter, etc...
-  company: Firma/Organisation
-  how_did_hear: Wie hast Du von uns erfahren?
-  subscribe: Registrieren
+  name: Naam
+  public_link: Sociale media link
+  public_link_placeholder: LinkedIn, Xitter, Mastadon, etc...
+  company: Werkgever
+  how_did_hear: Hoe heb je over ons gehoord? 
+  subscribe: Registeren
```
Comment (dometto) [inline _i18n/nl.yml:30]: typo

File: _i18n/nl.yml [hunk 4]
```diff
@@ -3,46 +3,51 @@ langs:
...
+    title: Toegangelijkheid
+    summary: We understand accessibility in a broad sense and strive to do better. We try to host meetings that are wheelchair accessible, smoke free, short(er) and provide child care. We provide information on the meeting location, including a description of the entrance, lighting and sounds, and policy around companion and service dogs beforehand so that participants can best plan for their visit. If you have feedback or questions regarding the accessibility of our meetings or initiatives, please get in touch at <a rel="noopener noreferrer" target="_blank" href="mailto:accessibility@techwerkers.nl">accessibility@techwerkers.nl</a>.
 
 press:
-  mentions: Pressespiegel
-  inquiry: Bitte kontaktieren Sie uns für Presseanfragen hier
+  mentions: Presse
```
Comment (dometto) [inline _i18n/nl.yml:42]:
```suggestion
  mentions: Pers
```

File: _i18n/nl.yml (hunk 3 above)
Comment (dometto) [reply _i18n/nl.yml:30]:
```suggestion
  subscribe: Registreren
```

File: _i18n/en.yml (hunk 1 above)
Comment (b-n) [reply _i18n/en.yml]: I assume Mastodon is the typo

**Pull request merged**